README.md

```diff
@@ -243,6 +243,10 @@ If an Observer fails to meet these expectations they can be excluded from future
 
 * [Community / Travel Fund](./community-fund/COMMUNITY_FUND_POLICY.md) - OpenJS Foundation projects and their community members are able to take advantage of our community fund.
 
+#### OpenJS Ecosystem Sustainability Program 
+
+* [OpenJS Ecosystem Sustainability Program](./project-resources/ESP/ECOSYSTEM_SUSTAINABILITY_PROGRAM.md) - The Ecosystem Sustainability Program is an opt-in OpenJS partner program to help secure outdated software and create new revenue streams for our projects and commercial participants.  
```

Comment:
```suggestion
* [OpenJS Ecosystem Sustainability Program](./project-resources/ESP/ECOSYSTEM_SUSTAINABILITY_PROGRAM.md) - The Ecosystem Sustainability Program is an opt-in OpenJS partner program to help secure outdated software and create new revenue streams for our projects.  
```
Creating revenue streams for commercial participants shouldn’t be one of our goals, imho.

Reply:
Without it, there won't be any new revenue streams for our projects, so i'm not sure how they can be separated.

Reply:
My point is that it shouldn’t be a goal for the foundation. Let’s say that tomorrow a philanthropy shows up and wants to do a partnership where 100% of the revenue goes to the projects and the foundation. we shouldn’t care that there isn’t a revenue stream for them.